doc/upload.md

```diff
@@ -67,6 +67,12 @@ spec:
 EOF
 ```
 
+### kubectl port-forward
```

Comment:
should be `Bare Metal` (or any other better suggestion)

Reply:
I've changed the header to something more descriptive.